src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -154,7 +153,7 @@ export class ServicesComponent extends BaseComponent<IServicesComponentPropertie
             id: ageKey,
             name: Resources.AgeText,
             minWidth: 150,
-            width: new ObservableValue(300),
+            width: -100,

```

Comment:
why -100, any specific reason not to use 150? as minwidth is 150.

Reply:
Negative widths are interpreted as percentages. Since we want the table columns to occupy full available width, I have set -100 which is equivalent to 100%.

Reply:
please add that comment for one of the columns, so in future engineers can know about it and can use this pattern if needed.